bin/yb-ctl

```diff
@@ -497,6 +514,10 @@ class ClusterOptions:
                                    "separated by commas.")
             self.placement_info.append(t_item)
 
+        self.num_drives = getattr(args, "num_drives", None)
```

Comment:
Yeah, it throws `AttributeError: 'Namespace' object has no attribute 'num_drives'` otherwise for commands that do not have `num_drives` setup, which i was trying to avoid.